src/js/media-chrome-range.js

```diff
@@ -130,7 +133,7 @@ template.innerHTML = `
       background-color: #777;
     }
   </style>
-  <input id="range" type="range" min="0" max="1000" step="1" value="0">
+  <input id="range" type="range" min="0" max="1000" step="any" value="0">
```

Comment:
I didn't know "any" was an option. It's possible this could simplify some of the other logic, where max is being converted to 1000 for more step granularity.